airflow/dag_processing/manager.py

```diff
@@ -514,6 +514,10 @@ def _deactivate_stale_dags(self, session=None):
                 if deactivated:
                     self.log.info("Deactivated %i DAGs which are no longer present in file.", deactivated)
 
+                for dag_id in to_deactivate:
+                    SerializedDagModel.remove_dag(dag_id)
+                    self.log.info("Deleted DAG %s in serialized_dag table", dag_id)
+
```

Comment:
Serialized DAGs for Deleted dag files are removed in the following place too

https://github.com/apache/airflow/blob/d8f8f8d3778fac8052862435e4e348f1f76f20ce/airflow/dag_processing/manager.py#L717-L722 

Does that not suffice?

Reply:
@kaxil `remove_deleted_dags` only removes dags whose dag_files are deleted.

this pr takes care of the case like a dag_id is renamed in a dag_file, or some dag_ids are removed in a dag_file

Reply:
Yup I know that :) -- What I more meant was do you think we should combine that or not run that now in Dag Processing Manager now that we have the changes in this PR? Don't see a point in having both the methods when one  in the PR is superset of the other.